doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -48,7 +48,7 @@ Build system
 nRF Front-End Modules
 ---------------------
 
-|no_changes_yet_note|
+* Added a clarification that the Simple GPIO implementation is intended for multiple front-end modules (not just one specific device) in the :ref:`ug_radio_fem_sw_support_mpsl` section of the :ref:`ug_radio_fem` guide.
```

Comment:
nRF front-end modules are no longer under "Application development" and the changelog tries to reflect the current documentation structure. Therefore, suggesting that you add a new heading "Working with RF front-end modules" right after the other Working with... headings and place this entry under it.

Reply:
In the "Application development" section there also is a "Build system" subsection which will need to be removed. Should I create another section for build system somewhere else on top level?

Reply:
I think we can remove the Build system heading for now.

Reply:
done